src/vs/workbench/browser/parts/editor/singleEditorTabsControl.ts

```diff
@@ -19,39 +19,98 @@ import { toDisposable } from '../../../../base/common/lifecycle.js';
 import { defaultBreadcrumbsWidgetStyles } from '../../../../platform/theme/browser/defaultStyles.js';
 import { IEditorTitleControlDimensions } from './editorTitleControl.js';
 import { BreadcrumbsControlFactory } from './breadcrumbsControl.js';
+import { IHoverOptions } from '../../../../base/browser/ui/hover/hover.js';
+import { IHoverDelegateOptions } from '../../../../base/browser/ui/hover/hoverDelegate.js';
+import { IConfigurationService } from '../../../../platform/configuration/common/configuration.js';
+import { WorkbenchHoverDelegate, IHoverService } from '../../../../platform/hover/browser/hover.js';
+import { IContextKeyService } from '../../../../platform/contextkey/common/contextkey.js';
+import { IContextMenuService } from '../../../../platform/contextview/browser/contextView.js';
+import { IInstantiationService } from '../../../../platform/instantiation/common/instantiation.js';
+import { IKeybindingService } from '../../../../platform/keybinding/common/keybinding.js';
+import { INotificationService } from '../../../../platform/notification/common/notification.js';
+import { IQuickInputService } from '../../../../platform/quickinput/common/quickInput.js';
+import { IThemeService } from '../../../../platform/theme/common/themeService.js';
+import { IEditorPartsView, IEditorGroupsView, IEditorGroupView } from '../../../../workbench/browser/parts/editor/editor.js';
+import { IReadonlyEditorGroupModel } from '../../../../workbench/common/editor/editorGroupModel.js';
+import { IEditorResolverService } from '../../../../workbench/services/editor/common/editorResolverService.js';
+import { IHostService } from '../../../../workbench/services/host/browser/host.js';
+import { MarkdownString } from '../../../../base/common/htmlContent.js';
 
 interface IRenderedEditorLabel {
 	readonly editor?: EditorInput;
 	readonly pinned: boolean;
 }
 
+class SingleEditorTabHoverDelegate extends WorkbenchHoverDelegate {
+
+	constructor(
+		private readonly control: SingleEditorTabsControl,
+		@IConfigurationService configurationService: IConfigurationService,
+		@IHoverService hoverService: IHoverService,
+
+	) {
+		super(
+			'element',
+			true,
+			(options) => {
+				return { ...this.getOverrideOptions(options), appearance: { showPointer: true } };
```

Comment:
Regarding `element` & `showPointer`, we tried this about half a year ago but it did not really feel very nice so we reverted that behaviour. I understand that this makes it easier to click the button, but I think users will still press even though it is further away.

Reply:
OK